ONSdigital/eq-questionnaire-launcher
# Fix mock SDS in test environment

### What is the context of this PR?
Added if statement that checks for the test launcher url before calling the SDS service so that the error isn't caused in the test environment.

### How to review
Using the other 2 PRs related to this (https://github.com/ONSdigital/eq-terraform-gcp-launcher/pull/17/files https://github.com/ONSdigital/eq-pipelines/pull/612/files) which add a new environment variable to specify whether the SDS is available, spin up a copy of the test environment which includes the changes in these PRs
- Create a GCP project
- Create a pipeline that specifies the project id
In the pipeline var file:
- Update the slack webhooks to post to the "personal" channel
- Update the monitoring alerting channel to the "personal" channel
- Update the fqdn values
In the pipeline config file:
- Update the "questionnaire-launcher-image" to point to the "fix-mocksds-in-test" tag
- Update the "eq-terraform-gcp-launcher" repo to point to the "introduce-env-var-for-sds-unavailible" branch
Once the environment is spun up test launching some non SDS schemas to make sure they still launch correctly

Do the same for a staging environment which will have SDS available and test that SDS schemas can still be launched
